Patterns/English/English-DateTime.yaml

```diff
@@ -477,7 +477,7 @@ LessThanRegex: !simpleRegex
 MoreThanRegex: !simpleRegex
   def: \b(more\s+than)\b
 DurationUnitRegex: !nestedRegex
-  def: (?<unit>{DateUnitRegex}|h(ou)?rs?|h|min(ute)?s?|sec(ond)?s?)\b
+  def: (?<unit>{DateUnitRegex}|h(ou)?rs?|h|min(ute)?s?|sec(ond)?s?|nights?)\b
```

Comment:
Why not make this change in DateUnitRegex? It's conceptually right and where similar cases for "days" are captured.

Reply:
Because then expressions like "last night" will be resolved as "daterange" instead of "datetimerange"

Reply:
Seems reasonable. But ideally the term should then be added closes to {DateUnitRegex} in the regex. Not a merge blocker though.
Thanks.